src/lua/loaders.lua

```diff
@@ -520,6 +520,31 @@ _G.require = function(modname)
     return raw_require(modname)
 end
 
+-- Load one of the given modules.
+--
+-- It just calls `require()` on the provided arguments (in the
+-- given order) and once this call succeeds, the function returns
+-- its result.
+--
+-- If neither of the given modules can be `require`'d, an error is
+-- raised.
+local function require_any(...)
```

Comment:
IMO, `any` is not the best choice, considering the semantic. Something like `require_first` or `require_next` is a bit better.

Reply:
Not a problem, changed to `require_first`.